README.md

```diff
@@ -71,6 +71,13 @@ We apply a custom theme to the Mantine components. If you are building a separat
 
  - Follow the **Hooks Order** as much as possible.
 
+## Guide: how to add a new panel
+- Create a panel and place it in the panels folder.
+- Include the panel in `src/windowmanagement/data/LazyLoads.tsx` and `src/windowmanagement/data/MenuItems.tsx`. This will make it appear in the menus.
+- We also need to add the panel to the OpenSpace Launcher. To do this, save a layout file by clicking `View > Save Task Bar Settings` and overwrite the JSON file at `OpenSpace > data > assets > web > default_ui_panels.json`.
+- This change in the engine needs to be committed to an engine branch.
+- Done!
```

Comment:
```suggestion
  - Create a panel and place it in the panels folder.
  - Include the panel in `src/windowmanagement/data/LazyLoads.tsx` and `src/windowmanagement/data/MenuItems.tsx`. This will make it appear in the menus.
  - We also need to add the panel to the OpenSpace Launcher. To do this, save a layout file by clicking `View > Save Task Bar Settings` and overwrite the JSON file at `OpenSpace > data > assets > web > default_ui_panels.json`.
  - This change in the engine needs to be committed to an engine branch.
  - Done!
```